packages/components/tag/src/tag.vue

```diff
@@ -58,9 +59,18 @@ const containerKls = computed(() => {
   ]
 })
 
+const isVisible = ref(true)
+
 // methods
 const handleClose = (event: MouseEvent) => {
-  emit('close', event)
+  if (props.disableTransitions) {
+    emit('close', event)
+  } else {
+    isVisible.value = false
```

Comment:
There is a problem here. The tag component can only emit the `close` event internally, and whether the component is deleted is controlled by external logic.

In this case, we seem to have no way to add a transition effect when the component disappears.

Reply:
In the example of the document, splice is used to delete the tag component, resulting in transition being unable to listen to the tag component uninstallation and animation, so an intermediate variable visible is added to ensure that the removal animation takes effect, and setTimeout(() => {
      emit('close', event)
    }, 0), guaranteed to trigger close